src/components/auth/poolData.js

```diff
@@ -1,8 +1,8 @@
 import { CognitoUserPool } from "amazon-cognito-identity-js";
 
 const poolData = {
-    UserPoolId: "us-east-2_iRsS9EAPT",
-    ClientId: "2pklf704q8vni1ifm7g4ohpup8",
+    UserPoolId: "us-east-2_4YSyhv0yv",
+    ClientId: "75ko0feo5lalfnakq1uhsviuks",
```

Comment:
I know you said not to worry about this but since this is a public repo, these are now compromised. We should make adding these to .env a priority. When you do, make sure to refresh these so that the exposed ones are invalid.

Reply:
I'll implement this on the next ticket.  Not too worried about these being public anyways. These are all burner user pools.